### PR TITLE
added some entries, other suggestions

### PR DESCRIPTION
**Pull request by bethsheets**
I added: WES, GA4GH, FAIR, API, interoperability

Other ideas things to consider that I don't have good definitions for... 
- Descriptor file
- parameter file
- Dockstore Legacy Registration
- Dockstore Legacy Tool
- Dockstore GitHub App
- Dockstore "Launch-With"
- Tool vs Workflow
- cloud costs? egress,  compute, preemptible
- local vs cloud computing?
- immutable
- Dockstore verified

I also suggest moving away from everything being Docker and instead focus on containerization, container, image. And then say Docker is one technology. We may also want to include Singularity and other docker-alternatives for HPCs?

### CODE REVIEW

File: entries.py [hunk 1]
```diff
@@ -18,7 +34,7 @@
 	pronunciation="", 
 	seealso="Biodata Catalyst")
 
-BDCat = GlossEntry("BDCat", 
+BDCat = GlossEntry("BD Catalyst",
```
Comment (aofarrel) [inline entries.py]: I've heard BDCat several times, but I've almost never heard BD Catalyst. Which one is more common in your experience?

File: entries.py [hunk 2]
```diff
@@ -10,6 +10,22 @@
...
+	acronym_full="Application Programmer Interfaces", 
+	definition="Application Programmer Interface technologies serve as software-based intermediaries to exchange data.", 
+	furtherreading="", 
+	institute="", 
+	pronunciation="", 
+	seealso="")
+
+AWS = GlossEntry("AWS", 
+	acronym_full="Amazon Web Services", 
+	definition="A provider of cloud services available on-demand.",
```
Comment (aofarrel) [inline entries.py]: Is there any information we should include here to contrast AWS against GCP?

File: entries.py [hunk 3]
```diff
@@ -146,6 +162,22 @@
...
+	acronym_full="", 
+	definition="A set of guidelines to improve the Findability, Accessibility, Interoperability, and Reuse of digital assets", 
+	furtherreading="https://www.go-fair.org/fair-principles/",
+	institute="", 
+	pronunciation='', 
+	seealso="")
+
+GA4GH = GlossEntry("GA4GH", 
+	acronym_full="Global Alliance For Genomics and Health", 
+	definition="The Global Alliance for Genomics and Health aims to accelerate progress in genomic research and human health by cultivating a common framework of standards and harmonized approaches for effective and responsible genomic and health-related data sharing.",
```
Comment (aofarrel) [inline entries.py:184]: Is there a way to "dictionaryafy" this by making it into a fragment? Would it be correct to call GA4GH "a network of public and private institutions which aims to accelerate..." or perhaps "a coalition of several institutions which aims to accelerate..."

File: entries.py [hunk 4]
```diff
@@ -234,6 +274,11 @@
 	pronunciation="", 
 	seealso="")
 
+WES = GlossEntry("Workflow Execution Service",
+	furtherreading="",
+	definition="A standardized API developed by [GA4GH] for describing a standard programmatic way to run and manage workflows.",
```
Comment (aofarrel) [inline entries.py:332]: What are some examples of platforms that can utilize this API?

**Review by aofarrel — CHANGES_REQUESTED** [review]

File: entries.py (hunk 1 above)
Comment (bethsheets) [reply entries.py]: BDCat is not an allowable abbreviation per the NHLBI style guide. Only BDC and BD Catalyst.

File: entries.py (hunk 2 above)
Comment (bethsheets) [reply entries.py]: Hmm, not sure how to be succinct with that. They offer different instance types and may have different cloud costs?

File: entries.py (hunk 3 above)
Comment (bethsheets) [reply entries.py:184]: I think either of those options is fine.

File: entries.py (hunk 4 above)
Comment (bethsheets) [reply entries.py:332]: The new Dockstore CLI integration with AWS Genomics utilizes WES.

Comment (aofarrel) [reply entries.py:332]: Actually, here's a better question: In 1.12 Dockstore we worked with Amazon to launch workflows in AWS via WES via the Dockstore API. Do you know if other platforms such as Terra and SB use WES too?

edit: Oops, forgot to refresh so I didn't see your reply, sorry to ask the same question twice

Comment (bethsheets) [reply entries.py:332]: I don't think so, but we should confirm on the Dockstore slack? I believe you can share run logs, etc using WES and I don't believe we get that with either of those launch-with partners at the moment.

File: entries.py (hunk 1 above)
Comment (aofarrel) [reply entries.py]: Is the NHLBI style guide public? Might be a good idea to link it somewhere (probably not in this document, but maybe elsewhere, or in our internal docs).

File: entries.py (hunk 4 above)
Comment (aofarrel) [reply entries.py:332]: It looks like the Dockstore CLI and maybe Cromwell are the only current public examples, so I think this definition is sufficient.

**Pull request merged**